pkg/loki/config_wrapper.go

```diff
@@ -0,0 +1,81 @@
+package loki
```

Comment:
can't comment on the file name, so i comment on line `1`, there's a typo in the file name.